shell/shared/shell.js

```diff
@@ -386,6 +386,13 @@ if (Meteor.isClient) {
     }
   });
 
+  Template.referrals.helpers({
+    setDocumentTitle: function () {
```

Comment:
Not a problem, just a remark:

I had forgotten this last time. Thanks to (I believe it was) @zarvox for noting it.